src/core/ledger/testUtils.js

```diff
@@ -29,3 +34,63 @@ export function createUuidMock(): UuidMock {
 
   return {setNextUuid, resetFakeUuid};
 }
+
+export const uuidMock: UuidMock = createUuidMock();
+
+const {resetFakeUuid, setNextUuid} = uuidMock;
+
+// Helper for constructing Grain values.
+export const g = (s: string): G.Grain => G.fromString(s);
+
+let nextFakeDate = 0;
+export function resetFakeDate() {
+  nextFakeDate = 0;
+}
```

Comment:
So there's an issue here where we have a piece of module state that's mutating across tests. Thus, we could get nondeterministic output depending on the order that the different test files execute in.

Basically, setting `nextFakeDate = 0` in the outer scope was kinda sketchy when this was inside a test describe block, but it's even sketchier when we factor it out into a module.

Can we wrap this state up in a closure like with `createUuidMock`